terra/terra/urls.py

```diff
@@ -126,11 +126,15 @@
     path("", views.index),
 ]
 
-urlpatterns += docs_urls
-urlpatterns += [path("storage/", include("storage.urls"))]
-urlpatterns += [path("tasks/", include("tasks.urls"))]
-urlpatterns += [path("estimators/", include("estimators.urls"))]
-urlpatterns += [path("credits/", include("credits.urls"))]
-urlpatterns += [path("admin/django-rq/", include("django_rq.urls"))]
-urlpatterns += [path("quotas/", include("quotas.urls"))]
-urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+urls_list += docs_urls
+urls_list += [path("storage/", include("storage.urls"))]
+urls_list += [path("tasks/", include("tasks.urls"))]
+urls_list += [path("estimators/", include("estimators.urls"))]
+urls_list += [path("credits/", include("credits.urls"))]
+urls_list += [path("admin/django-rq/", include("django_rq.urls"))]
```

Comment:
Let's leave this outside the `/v1` namespace. This is an endpoint related to the django admin, not the API.

Reply:
Ok. You mean line 134 or all this 4 lines?